Library/ViewModels/ChangePasswordViewModelTests.swift

```diff
@@ -145,73 +141,118 @@ final class ChangePasswordViewModelTests: TestCase {
   }
 
   func testValidationErrors_VoiceOverON() {
-    self.vm.inputs.viewDidAppear()
-
     let isVoiceOverRunning = { true }
 
     withEnvironment(isVoiceOverRunning: isVoiceOverRunning) {
-      self.vm.inputs.currentPasswordFieldDidReturn(currentPassword: "password")
-      self.vm.inputs.newPasswordFieldDidReturn(newPassword: "12345")
-      self.vm.inputs.newPasswordConfirmationFieldDidReturn(newPasswordConfirmed: "1234567")
+      self.vm.inputs.viewDidAppear()
+      self.validationErrorLabelIsHidden.assertLastValue(true)
+      self.validationErrorLabelMessage.assertLastValue("")
 
-      self.validationErrorLabelIsHidden.assertValues([false])
-      self.validationErrorLabelMessage
-        .assertValues(["Your password must be at least 6 characters long."])
-      self.accessibilityFocusValidationErrorLabel.assertValueCount(1)
-      self.saveButtonIsEnabled.assertValues([false])
+      self.vm.inputs.currentPasswordFieldTextChanged(text: "password")
+      self.accessibilityFocusValidationErrorLabel.assertValueCount(0)
+      self.saveButtonIsEnabled.assertValueCount(0)
+      self.validationErrorLabelIsHidden.assertLastValue(true)
```

Comment:
I find that using `assertValues` can help uncover instances where the signal is firing more often than it should be.

Reply:
In this particular example that's exactly what I was going to hide since the signals will be triggered multiple times but the final outcome is that the label will be hidden. But I can see the value of having historical values...will change that.